.gitmodules

```diff
@@ -1,6 +1,8 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/NOAA-EMC/spack.git
-  branch = jcsda_emc_spack_stack
+  ##url = https://github.com/spack/spack
+  ##branch = develop
+  #url = https://github.com/NOAA-EMC/spack.git
+  #branch = jcsda_emc_spack_stack
+  url = https://github.com/climbfuji/spack.git
+  branch = bugfix/parallelio_mpich_401
```

Comment:
Do you intend to point this at your fork?

Reply:
Temporary, need bugfixes for a few packages. Once the spack PR is ready and merged, I will revert.